app/controllers/support/providers_controller.rb

```diff
@@ -17,6 +17,10 @@ def new
       @provider.organisations.build
     end
 
+    def new_provider
```

Comment:
I think we should try to stick to RESTful actions. Are these methods just alternatives to their counterparts for the feature flag? I'd prefer we just make a new controller (since it will eventually become the main one)

Reply:
Done